crates/wdl-engine/tests/tasks.rs

```diff
@@ -88,7 +88,20 @@ fn run_test(test: &Path, config: TestConfig) -> BoxFuture<'_, Result<()>> {
             bail!("parsing failed: {e:#}");
         }
         if result.document().has_errors() {
-            bail!("test WDL contains errors; run a `check` on `source.wdl`");
+            let errors: Vec<_> = result
```

Comment:
I just found this easier to debug issues so that we don't have to do an out-of-band process to get the errors out (particularly difficult with feature flags).